content/en/integrations/guide/source-code-integration.md

```diff
@@ -220,7 +220,7 @@ Reporting commit 007f7f466e035b052415134600ea899693e7bb34 from repository git@gi
 
 {{< tabs >}}
 {{% tab "Error Tracking" %}}
-You can directly access a trace in its source repository on GitHub in [Error Tracking][1].
+You can directly access a trace in its source repository on the Git provider in [Error Tracking][1].
```

Comment:
- What do you think about removing `on the Git provider` entirely ? I'm not sure this term is widely used and understood
- Could be nice to rephrase, since "trace" isn't clearly referring to a stack trace (especially in the context of distributed tracing) - and links are tied to a specific frame
```suggestion
You can see links from stack frames to their source repository in [Error Tracking][1].
```